integration-tests/tests/e2e.rs

```diff
@@ -108,10 +109,10 @@ async fn test_e2e_impl<F>(verification_generator: F, diun_key_type: &str) -> Res
 where
     F: Fn(&TestContext) -> Result<(&'static str, String, &'static str)>,
 {
+    let ci = env::var("FDO_TEST_MODE").is_ok();
```

Comment:
I'd call this `FDO_PRIVILEGED` since we're not even looking what's inside anyway